src/ignitus-UserInterfaceBook/InterfaceSideNavigation/constants.ts

```diff
@@ -125,6 +125,10 @@ export const allEdges: Edges[] = [
         title: 'Toggle Buttons',
         route: '/interface/toggleButtons',
       },
+      {
+        title: 'Overlay Share',
+        route: '/interface/overlayShare',
```

Comment:
Create a parent `Overlay` under `Molecules` and add the Hooray and share overlays as children.

Reply:
Bifurcated into children